src/garage/torch/algos/ppo.py

```diff
@@ -75,19 +80,20 @@ def __init__(self,
                          policy_ent_coeff=policy_ent_coeff,
                          use_softplus_entropy=use_softplus_entropy,
                          stop_entropy_gradient=stop_entropy_gradient,
-                         entropy_method=entropy_method)
+                         entropy_method=entropy_method,
+                         minibatch_size=minibatch_size,
+                         optimization_epochs=optimization_epochs)
 
         self._lr_clip_range = lr_clip_range
 
-    def _compute_objective(self, advantages, valids, obs, actions, rewards):
-        """Compute objective using surrogate value and clipped surrogate value.
+    def _compute_objective(self, advantages, obs, actions, rewards):
+        """Compute objective value.
 
         Args:
-            advantages (torch.Tensor): Expected rewards over the actions
-            valids (list[int]): length of the valid values for each path
+            advantages (torch.Tensor): Advantage value at each step.
```

Comment:
if you are going to document a method with Tensors, you need to document the shapes of those tensors. please either remove the documentation or add shape documentation.

Reply:
For the methods that has return value, I need to document return type and information.
(Because of pylint warning)
Can I put `pylint: disable` for these private method that has return value?

Reply:
I just put shape of tensor in documents